de.metas.business.rest-api/src/main/java/de/metas/rest_api/ordercandidates/request/JsonOLCandCreateRequest.java

```diff
@@ -200,14 +202,29 @@
 	@JsonInclude(Include.NON_NULL)
 	private LocalDate presetDateShipped;
 
+	@ApiModelProperty(value = "A C_Payment entry will be created based on the info here")
+	private JsonPaymentInfo paymentInfo;
+
+	@ApiModelProperty(value = "Specifies if the created order will be a normal Sales Order or a Prepaid Sales Order")
+	private OrderDocType orderDocType;
+
+	@ApiModelProperty(value = "Specifies the payment rule that will propagate to the created order")
+	private JSONPaymentRule paymentRule;
+
+	@ApiModelProperty(value = "Specifies the value for the sales rep that will propagate to the created order")
+	private String salesPartnerCode;
+
+	@ApiModelProperty(value = "Specifies the value for the shipper that will propagate to the created order")
+	private String shipper;
+
 	@JsonCreator
 	@Builder(toBuilder = true)
 	private JsonOLCandCreateRequest(
 			@JsonProperty("org") final JsonOrganization org,
 			@JsonProperty("externalLineId") final String externalLineId,
 			@JsonProperty("externalHeaderId") final String externalHeaderId,
-			@JsonProperty("dataSourceInternalName") final @NonNull String dataSourceInternalName,
-			@JsonProperty("dataDestInternalName") final @Nullable String dataDestInternalName,
+			@JsonProperty("dataSource") final @NonNull String dataSource,
```

Comment:
pls coordinate this name (`@JsonProperty`) change with @pgandert1963 and @metas-Kay (AFAIK it's a documented sample-case in docs.metasfresh.org)